web-console/knikubevirt/vm-details/vm-boot-order/vm-boot-order.md

```diff
@@ -0,0 +1,51 @@
+# VM details - boot order
+
+## Entering edit mode
+![VM details - Overview tab](img/B1-0-0.jpg)
```

Comment:
Hi @Ranelim I noticed we are still using the older blue in this part of the flow. In other areas listed below, such as the wireframe to edit the virtual machine boot order we are using the newer blue, which leads me to believe you are adopting the new blue as a standard. If that is the case, it would be good to update the pages across the flow with the new PF blue: https://www.patternfly.org/v4/design-guidelines/styles/colors